app/app/api/private-cloud/products/_operations/update.ts

```diff
@@ -52,7 +52,10 @@ export default async function updateOp({
   ].filter((usr): usr is User => Boolean(usr));
 
   proms.push(subscribeUsersToMautic(users, request.decisionData.cluster, 'Private'));
-  proms.push(sendEditRequestEmails(request, false, session.user.name));
+
+  if (request.decisionStatus === DecisionStatus.APPROVED) {
+    proms.push(sendRequestApprovalEmails(request));
+  }
```

Comment:
what if it is not approved?

Reply:
lines 32-35

`  if (request.decisionStatus === DecisionStatus.PENDING) {
    await sendEditRequestEmails(request, true, session.user.name);
    return OkResponse(request);
  }`